src/include/simeng/Config.hh

```diff
@@ -16,14 +16,15 @@ class Config {
    "Vector-Length: 512, Streaming-Vector-Length: 512}, Fetch: "                \
    "{Fetch-Block-Size: 32, Loop-Buffer-Size: 64, Loop-Detection-Threshold: "   \
    "4}, Process-Image: {Heap-Size: 10485760, Stack-Size: 1048576, Mmap-Size: " \
-   "10485760}, Simulation-Memory: {Size: 104857600}, Register-Set: "           \
-   "{GeneralPurpose-Count: 154, FloatingPoint/SVE-Count: 90, "                 \
-   "Predicate-Count: 17, Conditional-Count: 128, Matrix-Count: 2}, "           \
-   "Pipeline-Widths: {Commit: 4, FrontEnd: 4, LSQ-Completion: 2}, "            \
-   "Queue-Sizes: {ROB: 180, Load: 64, Store: 36}, Branch-Predictor: "          \
-   "{BTB-Tag-Bits: 11, Saturating-Count-Bits: 2, Global-History-Length: 10, "  \
-   "RAS-entries: 5, Fallback-Static-Predictor: 2}, LSQ-L1-Interface: "         \
-   "{Access-Latency: 4, Exclusive: False, Load-Bandwidth: 32, "                \
+   "10485760}, Register-Set: {GeneralPurpose-Count: 154, "                     \
+   "FloatingPoint/SVE-Count: 90, Predicate-Count: 17, Conditional-Count: "     \
+   "128, Matrix-Count: 2}, Pipeline-Widths: {Commit: 4, FrontEnd: 4, "         \
+   "LSQ-Completion: 2}, Queue-Sizes: {ROB: 180, Load: 64, Store: 36}, "        \
+   "Branch-Predictor: {BTB-Tag-Bits: 11, Saturating-Count-Bits: 2, "           \
+   "Global-History-Length: 10, RAS-entries: 5, Fallback-Static-Predictor: "    \
+   "2}, Memory-Hierarchy: {Cache-Line-Width: 256, L1-Data: {Access-Latency: "  \
+   "4, Size: 65536}, DRAM: {Access-Latency: 200, Size: 2684354560}}, "         \
```

Comment:
Simulation memory size was 104857600 and the L1 cache size is 65536, and as far as I can see for all tests L1 cache params are used for all instances `simeng::memory::Mem`. Are you sure that this decrement in size won't cause any problems with the test suite?

Reply:
This is the default config file used for simulation, not the test suite. The config files used for the test suite have been configured with appropriate memory. Additionally, test suite classes have been updated to use the DRAM option rather than L1-Data to avoid confusion